frame/evm/src/backend.rs

```diff
@@ -176,6 +187,39 @@ impl<'vicinity, T: Trait> ApplyBackend for Backend<'vicinity, T> {
 			}
 		}
 
+		#[cfg(storage_tracing)] {
+			if affected_memory.len() > 0 {
```

Comment:
Small grumble -- maybe better call it `affected_storage(s)` as this is about EVM storage, not EVM memory.

Reply:
Good point